src/test/frontend/db/query_react_test.cljs

```diff
@@ -39,6 +39,16 @@ adds rules that users often use"
                                         [?bp :block/name ?current-page]]}))))
       ":current-page input resolves to current page name")
 
+  (is (= []
+         (map :block/content
+              (custom-query {:inputs [:current-page]
+                             :query '[:find (pull ?b [*])
+                                      :in $ ?current-page
+                                      :where [?b :block/page ?bp]
+                                      [?bp :block/name ?current-page]]}
+                            {:current-page-fn nil})))
+      ":current-page input fails gracefully when not present")
```

Comment:
I found this confusing as it's not failing but rather not resolving the input. I'm happy to tweak this wording later